sections/homepage/Assets/Assets.tsx

```diff
@@ -1,160 +1,530 @@
-	padding: 16px;
+const AssetName = styled.div`
+	font-size: 18px;
+	color: ${(props) => props.theme.colors.common.primaryWhite};
 `;
 
-const AssetCardText = styled(Paragraph)`
-	font-family: ${(props) => props.theme.fonts.bold};
-	font-size: 16px;
-	text-transform: capitalize;
-	color: ${(props) => props.theme.colors.white};
-	margin: 0px 0px 0px 16px;
-	text-align: center;
+const AssetPrice = styled.div`
+	font-family: ${(props) => props.theme.fonts.mono};
+	align-self: flex-end;
+	font-size: 20px;
+	color: ${(props) => props.theme.colors.common.primaryWhite};
+	width: 120px;
+	padding-left: 5px;
 `;
 
-const AssetsImage = styled.img`
-	max-width: 500px;
-	width: 100%;
+const AssetDescription = styled.div`
+	font-size: 11px;
+	color: ${(props) => props.theme.colors.common.secondaryGray};
 `;
 
-const MobileImage = styled(AssetsImage)`
-	margin: 0 auto;
-	display: block;
-	margin-top: 50px;
-	margin-bottom: 80px;
-	${media.lessThan('sm')`
-		width: 100%;
-		margin-bottom: 60px;
-	`}
+const StatsValue = styled.div`
+	font-size: 13px;
+	color: ${(props) => props.theme.colors.common.secondaryGray};
+	letter-spacing: -0.04em;
+	white-space: pre;
+`;
+
+const StyledFlexDivRow = styled(FlexDivRow)`
+	margin: auto;
+	margin-top: 40px;
+	gap: 20px 20px;
+	width: 1160px;
+	flex-wrap: wrap;
+	justify-content: center;
+`;
+
+const StatsCard = styled(GridContainer)`
+	cursor: pointer;
+	grid-template-columns: repeat(2, auto);
+	width: 275px;
+	height: 140px;
+	background: linear-gradient(180deg, rgba(40, 39, 39, 0.5) 0%, rgba(25, 24, 24, 0.5) 100%);
+	box-shadow: 0px 2px 2px rgba(0, 0, 0, 0.25), inset 0px 1px 0px rgba(255, 255, 255, 0.1),
+		inset 0px 0px 20px rgba(255, 255, 255, 0.03);
+	border-radius: 15px;
+	padding: 16px 20px;
+
+	svg.bg {
+		position: absolute;
+		z-index: 10;
+		margin-top: 16px;
+		margin-left: -20px;
+		width: 275px;
+		height: 140px;
+	}
 `;
 
-export const Bullet = styled.span`
-	display: inline-block;
-	width: 8px;
-	height: 8px;
-	border-radius: 100%;
-	background: ${(props) => props.theme.colors.gold};
+const Container = styled.div`
+	margin-bottom: 140px;
+`;
+
+const StyledCurrencyIcon = styled(Currency.Icon)`
+	width: 45px;
+	height: 45px;
+	margin-right: 15px;
+`;
+
+const TabButtonsContainer = styled.div`
+	display: flex;
+	margin-top: 40px;
+	margin-bottom: 35px;
+	width: 150px;
+	height: 28px;
+	justify-content: center;
+	align-items: center;
+	border-radius: 134px;
+	background: #1d1d1d;
+	border: 1px solid rgba(255, 255, 255, 0.1);
+	box-shadow: inset 0px -1.34783px 0px rgba(255, 255, 255, 0.08),
+		inset 0px 9.43478px 10.7826px rgba(0, 0, 0, 0.25);
+`;
+
+const MarketSwitcher = styled(FlexDiv)<{ isActive: boolean }>`
+	cursor: pointer;
+	height: 28px;
+	font-size: 13px;
+	width: 88px;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	border-radius: ${(props) => (props.isActive ? '100px' : '134px')};
+	color: ${(props) => (props.isActive ? props.theme.colors.common.primaryWhite : '#787878')};
```

Comment:
Seems `#787878` is `secondaryGray`, probably best to use the theme colors where possible in case of a global change.

Reply:
Done.